src/LSST/control/PowerSystem.cpp

```diff
@@ -38,15 +39,27 @@ namespace m2cellcpp {
 namespace control {
 
 PowerSystem::PowerSystem() : _motor(MOTOR), _comm(COMM) {
+    LDEBUG("Creating PowerSystem");
     _fpgaIo = FpgaIo::getPtr(); // If FpgaIo hasn't been setup, now is a good time to find out.
     _eThrd.run();
+
+    auto func = [this]() {
+        while (_timeoutLoop) {
```

Comment:
Please help to comment in the code that this `while` loop will run forever or not.